packages/thirdweb/src/react/core/account/provider.tsx

```diff
@@ -53,9 +52,6 @@ export function AccountProvider(
       "AccountProvider: No address passed. Ensure an address is always provided to the AccountProvider",
     );
   }
-  if (!isAddress(props.address)) {
```

Comment:
Why are we removing this?

Reply:
Don't wanna throw and break the page on bad input here, should throw inside the queries instead